src/com/google/javascript/jscomp/InlineAndCollapseProperties.java

```diff
@@ -176,6 +179,12 @@ public Builder testAggressiveInliningOnly(Consumer<GlobalNamespace> globalNamesp
       return this;
     }
 
+    @CanIgnoreReturnValue
+    public Builder setAssumeStaticInheritanceIsNotUsed(boolean assumeStaticInheritanceIsNotUsed) {
+      this.staticInheritanceUsed = assumeStaticInheritanceIsNotUsed == false;
```

Comment:
Suggested replacement:
      this.staticInheritanceUsed = !assumeStaticInheritanceIsNotUsed;